src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -69,7 +69,7 @@ public function find(string $name, string $default = null, array $extraDirs = []
         }
 
         $command = '\\' === \DIRECTORY_SEPARATOR ? 'where' : 'command -v';
```

Comment:
instead of the proposed change, I'd suggest this:
```suggestion
        $command = '\\' === \DIRECTORY_SEPARATOR ? 'where' : 'command -v -- ';
```

Reply:
@nicolas-grekas missed a blank space?

Reply:
The space is added on the line below